src/js/jsx/sections/libraries/LibrariesPanel.jsx

```diff
@@ -175,7 +171,7 @@ define(function (require, exports, module) {
                         disabled={this.props.disabled}
                         onDoubleClick={this.props.onVisibilityToggle} />
                     <div className={containerClasses}>
-                        {containerContents}
+                        {this._containerContents()}
```

Comment:
Cleaner to set this to a variable at the top and just put the variable in braces here.